dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -58,7 +58,7 @@
       Applitools::STITCH_MODE[:css]
   end
 
-  @eyes.check_window(identifier, MATCH_TIMEOUT, false)
+  @eyes.check_window(identifier, MATCH_TIMEOUT)
```

Comment:
Do you know what this boolean argument was intended to be doing?

Reply:
In particular, I'm worried about whether this will work in the new version with `stitch_mode = none`. Have you been able to verify any of the specific scenarios that use this option? 

Looking at applitools, all the runs from this branch I can see appear to only have four tests in them, none of which are testing this code path: ![image](https://github.com/user-attachments/assets/773eda23-610b-4c18-85b9-cf020e3cf541)

 Are these the right runs to be looking at, or am I missing something?

Reply:
It should be a fulls eyes run, results here: https://eyes.applitools.com/app/test-results/00000251677612358866/?accountId=mG4DoRfv0ntEgI3XIYBulrKxlZqrHtGwbP3V103ojvLFU110

Those were me running a single eyes test on the branch as I debugged what was globally wrong.

The stuck "Running" ones appear to be a result of our "retry UI tests due to flakiness" strategy: https://github.com/code-dot-org/code-dot-org/pull/60640#issuecomment-2311481612

Reply:
Oh, and I had looked at the history of altering that false flag, and this is from memory, but it was added by Brendan on a past eyes_selenium upgrade, but I actually inspected the code at the time, and even then that parameter should have mapped to `fully` (but it was undocumented at the time), which makes it a full page screenshot. HOWEVER, in that older version eyes.force_full_page_screenshot takes precedence over `fully`, whereas now the reverse is true.

What I wasn't able to figure out by code archeology is why passing false to the (then undocumented) parameter was ever required. But at least today its not required to run the eyes suite to success.

Reply:
Fantastic, thank you! I also did some poking around and was also unable to detangle intent from implementation here; seeing the results in applitools is good enough for me!